src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader/functions.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+use Symfony\Component\DependencyInjection\Alias;
+use Symfony\Component\DependencyInjection\Argument\IteratorArgument;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\ExpressionLanguage\Expression;
+
+function import($resource, $type = null, $ignoreErrors = false)
+{
+    return PhpFileLoader::call(function($file, Definition $defaults) {
+        $this->import($resource, $type, $ignoreErrors, $file);
+        $this->setCurrentDir(dirname($file));
+    });
+}
+
+/**
+ * @return DefaultsConfigurator
+ */
+function _defaults()
+{
+    return PhpFileLoader::call(function($file, Definition $defaults) {
```

Comment:
This also leads to easy auto-completion in the file itself, as you just need to add `/* @var $this The\Thing */` at the top to have static analysis and autocompletion.